andela/wger-space
# #166185758 Add Circle CI Integrations

#### What does this PR do?
Get Circle CI integrated with Github, Slack, and Coveralls

#### Description of Task to be completed?
- Added Circle Ci as circleci_space to the space integrations channel in Slack
- Pointed Circle CI to the Github branches to check for tests passing and then pass on the report to Coveralls for coverage of the tests written.
- Added badges for Circle CI build and Coveralls to the Readme.

#### How should this be manually tested?
- Check for the circleci_space user in the space integrations Slack channel you will be able to find the changes are being tracked as per the Github pushes.
<img width="856" alt="Screen Shot 2019-05-28 at 16 21 41" src="https://user-images.githubusercontent.com/43015966/58481475-1c6b8800-8165-11e9-919b-3f21a2c20d1a.png">

- There are badges of the build for Circle CI passing alongside the badge for code coverage for Coveralls. This is well demonstrated in the image below;
<img width="979" alt="Screen Shot 2019-05-28 at 16 25 39" src="https://user-images.githubusercontent.com/43015966/58481646-78361100-8165-11e9-8d90-1eb3154a7ce4.png">

- If a PR is raised and it's conforming to all the checks, something like this should be the displayed result 
<img width="832" alt="Screen Shot 2019-05-28 at 16 49 05" src="https://user-images.githubusercontent.com/43015966/58483307-b7199600-8168-11e9-8405-7fac8de17e5c.png">

#### Any background context you want to provide?
This is currently testable on the branch ch-circle-ci-integrations-with-slack-166185758 but will be available for every branch off develop once merged.

#### What are the relevant pivotal tracker stories?
[#166185758](https://www.pivotaltracker.com/story/show/166185758)
